api/app/services/pdfs/expense_sheet/generator_service_helpers.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Pdfs
+  module ExpenseSheet
+    module GeneratorServiceHelpers
+      def self.safe_call(object, *args)
+        object = object.call(*args) if object.is_a? Proc
```

Comment:
I'd try not to touch the parameter given from the outside, even if it actually not modifies it from callers perspective.
```suggestion
        return object.call(*args) if object.is_a? Proc
```

But this is highly discussable...